pycon/sponsorship/models.py

```diff
@@ -172,13 +178,7 @@ def joblisting_text(self):
 
     @property
     def website_logo(self):
-        if self.sponsor_logo is None:
-            benefits = self.sponsor_benefits.filter(benefit__type="weblogo", upload__isnull=False)[:1]
-            if benefits.count():
-                if benefits[0].upload:
-                    self.sponsor_logo = benefits[0]
-                    self.save()
-        return self.sponsor_logo.upload
+        return self.weblogo
```

Comment:
On Thu, Jul 23, 2015 at 11:46 AM, Rebecca <notifications@github.com> wrote:

> > -        return self.sponsor_logo.upload
> > +        return self.weblogo
>
> typo?
>

​Thanks, you have sharp eyes.​